NumericSequence.java

```diff
@@ -1,37 +1,18 @@
-import java.util.Scanner;
-
 /**
  * The program determines that the user-entered sequence of integers is non-decreasing.
  * The user can enter the sequence as a command-line argument.
  * If the user doesn't enter the sequence as a command-line argument,
  * the program requests a sequence from the console.
  */
 public class NumericSequence {
-
   public static void main(String[] args) {
-    if (args.length != 0) {
-      sequence(args);
+    Sequence sequence = new Sequence();
+    ReadFromConsole read = new ReadFromConsole();
+    String[] arr = read.splitLine();
+    if (args.length == 0) {
```

Comment:
Create Sequence provider class which responsibility is checking command line arguments and asking for input if needed